bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/AssetDataModel.java

```diff
@@ -47,6 +49,7 @@ default String getId() {
      *
      * @since com.adobe.cq.wcm.core.components.models.datalayer 1.0.0
      */
+    @JsonProperty("repo")
```

Comment:
Is `repo` the asset URL or the URL of the repository (store) where the asset is stored?

Reply:
That was supposed to be `repo:path` (https://github.com/adobe/xdm/blob/master/docs/reference/external/repo/asset.schema.md#repopath)

Reply:
Done in f71cfef